Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -646,7 +646,7 @@ def is_usable(self, force_update=False):
         time.sleep(.7)
 
         try:
-            with Timeout(seconds=30, patch=False):
+            with Timeout(seconds=300, patch=False):
```

Comment:
This can definitely be 30 seconds -- even under load, terminate shouldn't take more than a few seconds at most.